### ISSUE TITLE
Cannot specify compression codec

### ISSUE THREAD

**Issue by Chocorean** [Infinidat/infi.clickhouse_orm#115]
First thanks for your work. It's so pleasant to have such a great project with a lot of documentation.

My problem is that your orm does not allow to specify the compression codec to use when creating a table, and I need to. I'll just use this version for now and wait to see if there is an update.

See the [Clickhouse documentation](https://clickhouse.yandex/docs/en/query_language/create/#column-compression-codecs) for details.

**Comment by emakarov** [comment]
as a quick fix, you can inherit from required engine and override `create_table_sql` method 
https://github.com/Infinidat/infi.clickhouse_orm/blob/develop/src/infi/clickhouse_orm/engines.py#L69

**Comment by Chocorean** [comment]
It did the job.
May I offer you a MR ?

**Comment by ishirav** [comment]
@Chocorean - yes please.
Just keep in mind that the codec feature is relatively new - from ClickHouse release 19.1.6, 2019-01-24. So we need a way to preserve backwards compatibility with older ClickHouse versions (maybe by ignoring the codec if the database version is too old).

**Comment by Chocorean** [comment]
I've tried to create a branch to push my changes and submit a merge request but I encounter a permission error (access denied for `git push origin codec-dev`).

**Comment by Chocorean** [comment]
As I'm correctly passing the tests right now, I'm waiting for a way to push my few changes. :)
Still getting an error for pushing

**Comment by ishirav** [comment]
You cannot push directly to this repository, please create a pull request.
See https://help.github.com/en/articles/creating-a-pull-request

Thanks!

**Comment by Chocorean** [comment]
I just cannot create a new branch. I feel so dumb right now
Don't I just need to push the new branch ?
```
$ git push origin codec-dev 
ERROR: Permission to Infinidat/infi.clickhouse_orm.git denied to Chocorean.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by ishirav** [comment]
Fixed in v1.2.0
